File: dungeons-1.21.11/src/main/java/fishmod/utils/config/ModMenu.java
package fishmod.utils.config;

import fishmod.features.FishModScreen;
import com.terraformersmc.modmenu.api.ConfigScreenFactory;
import com.terraformersmc.modmenu.api.ModMenuApi;

public class ModMenu implements ModMenuApi {
    public ConfigScreenFactory<?> getModConfigScreenFactory() {
        return parent -> new FishModScreen();
    }
}

File: dungeons-1.21.11/src/main/kotlin/fishmod/utils/config/ModMenu.kt
package fishmod.utils.config

import com.terraformersmc.modmenu.api.ConfigScreenFactory
import com.terraformersmc.modmenu.api.ModMenuApi
import fishmod.features.FishModScreen

class ModMenu : ModMenuApi {
    override fun getModConfigScreenFactory(): ConfigScreenFactory<*> {
        return ConfigScreenFactory { parent -> FishModScreen() }
    }
}
